packages/specs/schema/src/domain/JsonSchema.ts

```
@@ -63,29 +80,31 @@
+   * Internal custom props
+   * @private
+   */
+  #vendors = new Map<string, any>();
   #discriminator: null | Discriminator = null;
-  #genericLabels: string[];
-  #nestedGenerics: Type<any>[][] = [];
   #alias: AliasMap = new Map();
   #itemSchema: JsonSchema;
   #target: Type<any>;
-  #isGeneric: boolean = false;
   #isCollection: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)